lerna.json

```diff
@@ -0,0 +1,6 @@
+{
+  "version": "6.5.0",
+  "npmClient": "yarn",
+  "useWorkspaces": true,
+  "packages": ["packages/*", "plugins/*"]
```

Comment:
This is interesting... I've never really seen a monorepo not just dump everything in `packages`. Makes sense though.

Reply:
storybook does it this way and I am a fan of the seperation